<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveTreeItem</class>
 <widget class="QDialog" name="MoveTreeItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Перемещение</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Подразделение:</string>
       </property>
       <property name="buddy">
        <cstring>movedItem</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="movedItem">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineFilter"/>
         </item>
         <item>
          <widget class="QPushButton" name="findButton">
           <property name="text">
            <string>Найти</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/16x16/find/ico/find_16x16.png</normaloff>:/16x16/find/ico/find_16x16.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Переместить подразделение в:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTreeView" name="newParents">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineFilterWp"/>
         </item>
         <item>
          <widget class="QPushButton" name="findWpButton">
           <property name="text">
            <string>Найти</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/16x16/find/ico/find_16x16.png</normaloff>:/16x16/find/ico/find_16x16.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Переместить рабочее место в:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTreeView" name="newParentsWp">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="delWp">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Не перемещать рабочие места (удалить их)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="delDep">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Не перемещать подразделения (удалить их)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="moveVariants">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Варианты перемещения подразделения:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="fullItem">
        <property name="text">
         <string>Полностью подразделение</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="onlyChildrenItem">
        <property name="text">
         <string>Только дочерние подразделения</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>Применить</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/apply/ico/apply_16x16.png</normaloff>:/16x16/apply/ico/apply_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>movedItem</tabstop>
  <tabstop>lineFilter</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>newParents</tabstop>
  <tabstop>lineFilterWp</tabstop>
  <tabstop>findWpButton</tabstop>
  <tabstop>newParentsWp</tabstop>
  <tabstop>delWp</tabstop>
  <tabstop>delDep</tabstop>
  <tabstop>fullItem</tabstop>
  <tabstop>onlyChildrenItem</tabstop>
  <tabstop>moveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MoveTreeItem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delWp</sender>
   <signal>clicked(bool)</signal>
   <receiver>newParentsWp</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delWp</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineFilterWp</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delWp</sender>
   <signal>clicked(bool)</signal>
   <receiver>findWpButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>newParents</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineFilter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delDep</sender>
   <signal>clicked(bool)</signal>
   <receiver>findButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
